lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -0,0 +1,154 @@
+;(function (define, undefined) {
```

Comment:
Add a spec file to test this factory. It should verify that all the correct fields are created in the view. It should also verify what happens when the settings API returns a 500.